site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -148,30 +148,21 @@ Other optimization issues
 C++ exceptions
 --------------
 
-C++ exceptions are turned off by default in ``-O1`` (and above). This prevents the generation of ``try-catch`` blocks, which lets the code run much faster, and also makes the code smaller. 
+Catching C++ exceptions is turned off by default in ``-O1`` (and above). This prevents the generation of ``try-catch`` blocks, which lets the code run much faster, and also makes the code smaller. 
```

Comment:
I think it would be clearer to say that it removes `catch` blocks, since the content of the try block will still be there. And maybe also add that it turns throws into aborts/termination. And maybe make it clear that the huge speed gain is because of the particular way exceptions have to be lowered in asm.js and MVP wasm (rather than some intrinsic problem).

Reply:
Thanks, adding.